### PR TITLE
Allow specifying port through PAPERLESS_PORT environment variable

### PR DESCRIPTION
**Pull request by yschroeder**
Supersedes #25 and adresses the issue mentioned in https://github.com/jonaswinkler/paperless-ng/pull/1285#discussion_r706390519.

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -699,6 +699,12 @@ PAPERLESS_WEBSERVER_WORKERS=<num>
 
     Defaults to 2.
 
+PAPERLESS_PORT=<port>
+    The port number the webserver will listen on. Change the port if it collides
```
Comment (tribut) [inline docs/configuration.rst:703]: We should probably make it clear that this only changes the listening port *inside the countainer* and conflicts on the host should be resolved by modifying `docker-compose.yml`.

Comment (yschroeder) [reply docs/configuration.rst:703]: Sure, go ahead and add it. I'm no expert on docker, so you can likely find the better words for this.

Comment (tribut) [reply docs/configuration.rst]: I've changed the text. Let me know if you dislike this. Otherwise I'd say LGTM.

**Comment by yschroeder** [comment]
Your clarification looks good 👍

**Review by tribut — APPROVED** [review]

**Comment by morremeyer** [comment]
If you’re coming to this PR to find out why your installation in Kubernetes broke when upgrading to 1.6.0, please check #264.

**Pull request merged**